homeassistant/components/anthropic/config_flow.py

```diff
@@ -128,21 +132,50 @@ async def async_step_init(
     ) -> ConfigFlowResult:
         """Manage the options."""
         options: dict[str, Any] | MappingProxyType[str, Any] = self.config_entry.options
+        errors: dict[str, str] = {}
 
         if user_input is not None:
```

Comment:
Can we turn some of this code around, so handle `user_input is None` first, and then return to save on indentation ?

Reply:
ah I guess we always need that last block in both cases.